vulnerable_people_form/integrations/spl_check.py

```diff
@@ -8,11 +8,13 @@
 
 
 def check_spl(nhs_number, date_of_birth):
+    test_mode = 'NO'
     records = execute_sql(
-        "CALL cv_base.is_person_currently_on_the_spl(:nhs_number, :date_of_birth)",
+        "CALL cv_base.is_person_currently_on_the_spl_new(:nhs_number, :date_of_birth, :test_mode)",
```

Comment:
what does test_,mode mean?

Reply:
Linked with https://github.com/alphagov/covid-engineering/pull/1001

Still under review on db side... This is done because mysql doesnt support optional parameters... and we need to bypass transactions state setting in test mode.... hence the flag which toggles the same in stored procedure..